README.md

```diff
@@ -64,6 +64,8 @@ Minimum Requirements: macOS 11 (Big Sur), Windows 10, Ubuntu 18.04, Fedora 32, D
 
 This is a fork of the original project: [bsharper/atv-desktop-remote](https://github.com/bsharper/atv-desktop-remote)
 
+**What's New in This Fork?** See [CHANGES_IN_THIS_FORK.md](CHANGES_IN_THIS_FORK.md) for a comprehensive list of user experience improvements and developer enhancements that distinguish this fork from the original project.
```

Comment:
We should have some key highlights included in this README.md, with a link to the full document.

The purpose of this fork would be clearer if this section was listed earlier in the readme.